source/win-main/MainEditor.vue

```diff
@@ -572,7 +572,10 @@ export default defineComponent({
     mdEditor.on('zettelkasten-link', (linkContents) => {
       ipcRenderer.invoke('application', {
         command: 'force-open',
-        payload: linkContents
+        payload: {
+                linkContents: linkContents,
+                newTab: false // let open-file command decide based on preferences
```

Comment:
I would suggest to set this to `undefined`, which will perfectly work in the main process and furthermore express that we do not care whether or not this file will be opened in a new tab. Makes it easier to see.